ConfigFiles/PlusDeviceSet_Server_StealthLinkTracker.xml

```diff
@@ -59,6 +61,7 @@ Tools: Stylus (Passive Planar, Blunt) , Reference (Small Passive Cranial Frame)"
         <Message Type="TRANSFORM" />
       </MessageTypes>
       <TransformNames> 
+        <!-- Remove transforms for tools that are not defined by the current StealthLink server -->

```

Comment:
Add "To avoid ... warnings, ". Replace ... with a few words that appear in the error message so that users can recognize what warning you mean.

Reply:
I meant the warning that you get when no path found between From->To coordinate systems.